internal/providers/terraform/google/kms_crypto_key.go

```diff
@@ -0,0 +1,183 @@
+package google
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/usage"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetKMSCryptoKeyRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "google_kms_crypto_key",
+		RFunc: NewKMSCryptoKey,
+	}
+}
+
+func NewKMSCryptoKey(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	region := d.Get("region").String()
+
+	algorithm := "GOOGLE_SYMMETRIC_ENCRYPTION"
+	protectionLevel := "SOFTWARE"
+
+	if d.Get("version_template").Type != gjson.Null {
+		algorithm = d.Get("version_template.0.algorithm").String()
+		protectionLevel = d.Get("version_template.0.protection_level").String()
+	}
+
+	monthlyKeys := decimal.Zero
```

Comment:
This should default to `*decimal.Decimal`, since if usage-file.key_versions is not set AND rotation_period is not set (as it's optional in terraform), then this is shown:
```
resource "google_kms_crypto_key" "my_keys" {
  name            = "crypto-key-example"
  key_ring        = ""
}

results in:

  google_kms_crypto_key.my_keys
	├─ Key versions                                        0  months          0.0600       0.0000        0.0000
```
But `-` should be shown for the QTY and costs as we don't know the usage, so shouldn't guess the cost.

Reply:
fixed this in the latest commit